data/multiplayer/scenarios/ANL_utils/ANL_leader_options.cfg

```diff
@@ -369,7 +374,7 @@ Help an ally with their research")}
                                         [message]
                                             caption= _ "Diplomatic Options"
                                             speaker=unit
-                                            image=items/book4.png
+                                            image= "items/book4.png"
```

Comment:
Oh, sorry, I had seen it like that earlier, so I thought it was something I had messed up accidentally.
Does that mean I should change it back?